sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/aad/webapp/AuthorizationClientProperties.java

```diff
@@ -10,6 +10,8 @@
  */
 public class AuthorizationClientProperties {
 
+    public static final String AZURE = "azure";
```

Comment:
I mean delete this one, and use `AADClientRegistrationRepository.AZURE` instead.

Reply:
Got it!